lib/utils.js

```diff
@@ -1,13 +1,127 @@
-'use strict';
 
-exports.contains = function contains(str) {
+"use strict";
+
+var path = require("path")
+  , fs = require("fs")
+  , async = require("async")
+  , log = require("./log")
+  , mail = require("./mail");
+
+module.exports.contains = function contains(str) {
     return function(v) {
         return v.indexOf(str) >= 0;
     };
 };
 
-exports.startsWith = function startsWith(str) {
+module.exports.startsWith = function startsWith(str) {
```

Comment:
Oh I know - that code is a relic of another author, I'm slowly eradicating its usage.